DrawerKit/DrawerKit/Public API/Notifications.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+public class NotificationToken {
+    public let token: NSObjectProtocol
+    public let center: NotificationCenter
+
+    init(token: NSObjectProtocol, center: NotificationCenter) {
+        self.token = token
+        self.center = center
+    }
+    
+    deinit { center.removeObserver(token) }
+}
+
+
+public protocol NotificationEnum {
+    var name: Notification.Name { get }
+}
+
```

Comment:
ditto

Reply:
Fixed in https://github.com/Babylonpartners/DrawerKit/pull/36/commits/8645a1610ad9f6d81fd3ddcab8caa45412d61f4c.